composer.json

```diff
@@ -20,6 +20,7 @@
     },
     "require-dev": {
         "matthiasnoback/symfony-dependency-injection-test": "^2.0",
+        "phpunit/phpunit": "^6.5",
```

Comment:
I use simple-phpunit IIRC

Reply:
It does not work because phpunit missing.

Reply:
I've just checked:

```bash
▶ php ./vendor/bin/simple-phpunit 
PHP Fatal error:  Uncaught Exception: simple-phpunit requires the "zip" PHP extension to be installed and enabled in order to uncompress the downloaded PHPUnit packages. in /home/roukmoute/workspace/enum/vendor/symfony/phpunit-bridge/bin/simple-phpunit:73
Stack trace:
#0 {main}
  thrown in /home/roukmoute/workspace/enum/vendor/symfony/phpunit-bridge/bin/simple-phpunit on line 73
```

Reply:
What does `php -m|grep zip`return?

Reply:
empty

Reply:
That means the issue is with your system. Maybe it is installed but still not enabled?

Reply:
I don't think it currently available with php 7.2.

Reply:
Aren't there Travis builds that use 7.2, simple-phpunit and work on this repo?